datasets/hatexplain/README.md

```diff
@@ -0,0 +1,175 @@
+
+## Dataset Structure
+
+### Data Instances
+
+Sample Entry:
+~~~
+{
+  "24198545_gab": {
+    "post_id": "24198545_gab",
+    "annotators": [
+      {
+        "label": "hatespeech",
+        "annotator_id": 4,
+        "target": ["African"]
+      },
+      {
+        "label": "hatespeech",
+        "annotator_id": 3,
+        "target": ["African"]
+      },
+      {
+        "label": "offensive",
+        "annotator_id": 5,
+        "target": ["African"]
+      }
+    ],
+    "rationales":[
+    [0,0,0,0,0,0,0,0,1,0,0,1,1,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0],
+    [0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],
+    [0,0,0,0,0,0,0,0,1,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
+    ],
+    "post_tokens": ["and","this","is","why","i","end","up","with","nigger","trainee","doctors","who","can","not","speak","properly","lack","basic","knowledge","of","biology","it","truly","scary","if","the","public","only","knew"]
+  }
+}
+~~~
+
+### Data Fields
+
+:small_blue_diamond:post_id : Unique id for each post<br/>
+:small_blue_diamond:annotators : The list of annotations from each annotator<br/>
+:small_blue_diamond:annotators[label] : The label assigned by the annotator to this post. Possible values: [Hatespeech, Offensive, Normal]<br/>
+:small_blue_diamond:annotators[annotator_id] : The unique Id assigned to each annotator<br/>
+:small_blue_diamond:annotators[target] : A list of target community present in the post<br/>
+:small_blue_diamond:rationales : A list of rationales selected by annotators. Each rationales represents a list with values 0 or 1. A value of 1 means that the token is part of the rationale selected by the annotator. To get the particular token, we can use the same index position in "post_tokens"<br/>
+:small_blue_diamond:post_tokens : The list of tokens representing the post which was annotated<br/>
+
+### Data Splits
+
+[Post_id_divisions](https://github.com/punyajoy/HateXplain/blob/master/Data/post_id_divisions.json) has a dictionary having train, valid and test post ids that are used to divide the dataset into train, val and test set in the ratio of 8:1:1.
+
+
+
+## Dataset Creation
+
+### Curation Rationale
+
+[Needs More Information]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+[Needs More Information]
+
+#### Who are the source language producers?
+
+[Needs More Information]
+
+### Annotations
+
+#### Annotation process
+
+[Needs More Information]
+
+#### Who are the annotators?
+
+[Needs More Information]
+
+### Personal and Sensitive Information
+
+[Needs More Information]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[Needs More Information]
+
+### Discussion of Biases
+
+[Needs More Information]
+
+### Other Known Limitations
+
+[Needs More Information]
+
+## Additional Information
+
+### Dataset Curators
+
+[Needs More Information]
+
+### Licensing Information
+
+[Needs More Information]
+
+### Citation Information
+
+[Needs More Information]
```

Comment:
````suggestion
```bibtex
@misc{mathew2020hatexplain,
      title={HateXplain: A Benchmark Dataset for Explainable Hate Speech Detection}, 
      author={Binny Mathew and Punyajoy Saha and Seid Muhie Yimam and Chris Biemann and Pawan Goyal and Animesh Mukherjee},
      year={2020},
      eprint={2012.10289},
      archivePrefix={arXiv},
      primaryClass={cs.CL}
}
```
````

Reply:
Outdated suggestion. I will make a fresh commit